private_gpt/components/llm/llm_component.py

```diff
@@ -57,3 +57,11 @@ def __init__(self, settings: Settings) -> None:
                 self.llm = OpenAI(api_key=openai_settings)
             case "mock":
                 self.llm = MockLLM()
+
+            case "bedrock":
+                from llama_index.llms import Bedrock
+
+                self.llm = Bedrock(
```

Comment:
You could be passing the credentials for consistency, the same way you do in the embedding component

Reply:
done